src/backend/test/unit/pia-intake/pia-intake.service.spec.ts

```diff
@@ -136,6 +136,78 @@ describe('PiaIntakeService', () => {
         'updatedByDisplayName',
       ]);
 
+      expect(result).toEqual(getPiaIntakeRO);
+    });
+    it('succeeds calling the database and update submittedAt column when the drafter submit pia for', async () => {
+      const createPiaIntakeDto: CreatePiaIntakeDto = {
+        ...createPiaIntakeMock,
+        submittedAt: null,
+      };
+      const piaIntakeEntity = { ...piaIntakeEntityMock };
+      const getPiaIntakeRO = { ...getPiaIntakeROMock };
+
+      const user: KeycloakUser = { ...keycloakUserMock };
+
+      piaIntakeRepository.save = jest.fn(async () => {
+        delay(10);
+        return piaIntakeEntity;
+      });
+
+      omitBaseKeysSpy.mockReturnValue(getPiaIntakeRO);
+
+      const result = await service.create(createPiaIntakeDto, user);
+
+      expect(piaIntakeRepository.save).toHaveBeenCalledWith({
+        ...createPiaIntakeDto,
+        createdByGuid: user.idir_user_guid,
+        createdByUsername: user.idir_username,
+        updatedByGuid: user.idir_user_guid,
+        updatedByUsername: user.idir_username,
+        updatedByDisplayName: user.display_name,
+        drafterEmail: user.email,
+        submittedAt: new Date(),
+      });
+
+      expect(omitBaseKeysSpy).toHaveBeenCalledWith(piaIntakeEntity, [
+        'updatedByDisplayName',
+      ]);
+
+      expect(result).toEqual(getPiaIntakeRO);
+    });
+
+    it('succeeds calling the database and not update submittedAt column when the drafter already submit pia previous', async () => {
```

Comment:
Do we even need this test? Not sure if it is testing anything other than what is already being tested by existing tests cases.